lib/tasks/bonnie.rake

```diff
@@ -36,8 +36,13 @@ namespace :cqm do
         else
           hds_value_sets = []
           value_set_fixtures.each_pair do |_, versions|
-            versions.each_pair do |_, vs_json|
-              hds_value_sets << HealthDataStandards::SVS::ValueSet.new(vs_json)
+            # if there is a _id then this level is a actually a valueset fixture
+            if versions.key?('_id')
```

Comment:
if I understand correctly, the values set fixture file can be
an array of valuesets
a version to oid to valueset map
a version to valueset map
Maybe add a comment about the supported formats?

Reply:
Yeah. There are now three different types of value set fixtures, it's very frustrating. I'll add a comment to explain that.